src/compass/s1_cslc_qa.py

```diff
@@ -466,3 +478,97 @@ def write_qa_dicts_to_json(self, file_path):
         # write combined dict to JSON
         with open(file_path, 'w') as f:
             json.dump(output_dict, f, indent=4)
+
+
+    def compute_valid_land_pixel_percent(self, cslc_h5py_root, pol):
+        '''
+            Compute the ratio of valid pixels on land area
+
+            Parameters
+            ----------
+            cslc_h5py_path: h5py.File
+                Root of the CSLC-S1 HDF5 product
+            shapefile_path: str
+                Path to the coastline shapefile
+
+
+            Returns
+            -------
+            ratio_valid_pixel_land:  float
+                Ratio of valid pixels on land
+        '''
+
+        drv_shp_in = ogr.GetDriverByName('GPKG')
+        if os.path.exists(LAND_GPKG_FILE):
+            coastline_shapefile = drv_shp_in.Open(LAND_GPKG_FILE)
+        else:
```

Comment:
I think here it would be more appropriate to have a `try`/`except` instead of a `if`/`else`

Reply:
I think this ok. The rest of the code uses `if/else` for path checking.

Reply:
Will keep the code as it is.